packages/web3-wrapper/CHANGELOG.json

```diff
@@ -1,4 +1,14 @@
 [
+    {
+        "version": "3.1.1",
```

Comment:
should this be a major version bump, I can go either way

Reply:
In this case, I think we presented users with a pretty clear description of the intended behavior in the type definition and the documentation. The actual behavior did not match that intended behavior. I would classify this as a bug fix.

However I admit the decision is not always clear-cut. There's a [decent discussion on StackOverflow](https://softwareengineering.stackexchange.com/questions/368030/is-a-breaking-change-bug-fix-a-semver-patch-or-a-major) about it.